make_zipapp.sh

```diff
@@ -8,14 +8,16 @@ mkdir build/piman.app
     mv $(ls | grep -v -) ../../../piman.app
 )
 cp -r logging.conf utility tftp dhcp monitoring tcp *.py build/piman.app
+# copy hosts.csv and monitoring.config into build
+cp -r hosts.csv monitoring/monitoring.config build
 mkdir build/piman.app/install
 mkdir build/logs
 cp piman.yaml build
 #git clone --depth 1 --single-branch https://github.com/raspberrypi/firmware.git firmware
 if [ ! -f firmware/boot/bootcode.bin ]
 then
-    git submodule init firmware
-    git submodule update --depth 1 firmware
+   git submodule init firmware
```

Comment:
oops it looks like you deleted some spaces by accident.

Reply:
For the "install client script", do you mean by make_zipapp.sh or the .sh that starts the monitoring service?